layouts/partials/components/edit-url.html

```diff
@@ -0,0 +1,19 @@
+{{- $editThisPage := (T "editThisPage") | default "Edit this page"}}
+
+<div class="mt-12 mb-8 block text-xs text-gray-500 ltr:text-right rtl:text-left dark:text-gray-400">
```

Comment:
can you double check the classes are consistent with what we had previously in the `toc.html`?

Reply:
It should be fine now (and rebased)